skada/tests/test_reweight.py

```diff
@@ -207,7 +219,7 @@ def test_new_X_adapt(estimator, da_reg_datasets):
         DiscriminatorReweightAdapter(),
         KLIEPReweightAdapter(gamma=[0.1, 1, "auto", "scale"], random_state=42),
         KMMReweightAdapter(gamma=0.1, smooth_weights=True),
-        MMDTarSReweightAdapter(gamma=1.0),
+        # MMDTarSReweightAdapter(gamma=1.0),
```

Comment:
shoudl be decommented

Reply:
oh, yeah. that's the one that fails only when all tests are running but works if you run a single module

Reply:
this seems to be a problem with dataset generator. if i add

```
    assert not np.any(np.isnan(y_train))
```

to check that we don't get `NaN` labels, many of those tests fail. i'm gonna add separate issue for this. seems completely irrelevant for this PR. wdyt?

Reply:
OK seems irrelevant indeed

Reply:
but we must open a issus to add thsoe etst back